frontend/src/components/Thread/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FunctionComponent, CSSProperties } from 'react';
+import PartisipantsList from '../ParticipantsList';
+import TextEditor from '../TextEditor';
+import styles from './styles.module.sass';
+import { IUser } from '../../common/models/user';
+
+interface IProps {
+  width: number | string;
+  participants: IUser[];
+  // comments;
+}
+
+const createMessage = (text = '') => <div className={styles.mockMessage}>{text}</div>;
+
+const Thread: FunctionComponent<IProps> = ({ width, participants }) => {
+  const usNamesList = participants.map(us => us.fullname);
```

Comment:
Comment if u don't use it now.
We can pass users to the participants list to prevent unnecessary calculations.